cmd/security-agent/api/agent/agent.go

```diff
@@ -119,3 +121,14 @@ func makeFlare(w http.ResponseWriter, r *http.Request) {
 	}
 	w.Write([]byte(filePath))
 }
+
+func getRuntimeConfig(w http.ResponseWriter, r *http.Request) {
+	runtimeConfig, err := yaml.Marshal(config.Datadog.AllSettings())
+	if err != nil {
+		log.Errorf("Unable to marshal runtime config response: %s", err)
+		body, _ := json.Marshal(map[string]string{"error": err.Error()})
+		http.Error(w, string(body), 500)
+		return
+	}
+	w.Write(runtimeConfig)
```

Comment:
Hey @xornivore ,
Could you add some code to avoid returning sensitive credentials ?
Looking at what the agent does [here](https://github.com/DataDog/datadog-agent/blob/master/cmd/agent/api/agent/agent.go#L286), it seems that `log.CredentialsCleanerBytes` should do the trick.

Reply:
Ah good point, thank you!